lms/templates/registration/password_reset_confirm.html

```diff
@@ -3,148 +3,70 @@
 
 {% block title %}
 <title>
-  {% blocktrans with platform_name=platform_name %}
-      Reset Your {{ platform_name }} Password
-  {% endblocktrans %}
+{% blocktrans with platform_name=platform_name %}
+    Reset Your {{ platform_name }} Password
+{% endblocktrans %}
 </title>
 {% endblock %}
 
 {% block bodyextra %}
-  <script type="text/javascript">
-    $(function() {
-
-      // adding js class for styling with accessibility in mind
-      $('body').addClass('js');
-
-      // new window/tab opening
-      $('a[rel="external"], a[class="new-vp"]')
-      .click( function() {
-      window.open( $(this).attr('href') );
-      return false;
-      });
-
-      // form field label styling on focus
-      $("form :input").focus(function() {
-        $("label[for='" + this.id + "']").parent().addClass("is-focused");
-      }).blur(function() {
-        $("label").parent().removeClass("is-focused");
-      });
-    });
-  </script>
+<script type="text/javascript" src="{{STATIC_URL}}js/student_account/password_reset.js"></script>
 {% endblock %}
 
 {% block bodyclass %}view-passwordreset{% endblock %}
 
 {% block body %}
-  <section class="introduction">
-    <header>
-      <h1 class="title">
-        <span class="title-super">
-          {% blocktrans with platform_name=platform_name %}
-          Reset Your {{ platform_name }} Password
-          {% endblocktrans %}
-        </span>
-      </h1>
-    </header>
-  </section>
-
-  <section class="passwordreset container">
-    <section role="main" class="content">
-      {% if validlink %}
-      <header>
-      <h2 class="sr">{% trans "Password Reset Form" %}</h2>
-      </header>
-
-      <form role="form" id="passwordreset-form" method="post" action="">{% csrf_token %}
-        <!-- status messages -->
-        <div role="alert" class="status message">
-          <h3 class="message-title">
-            {% blocktrans %}
-                We're sorry, but this version of your browser is not supported. Try again using a different browser or a newer version of your browser.
-            {% endblocktrans %}
-          </h3>
-        </div>
-
-        {% if err_msg %}
-        <div role="alert" class="status message submission-error" style="display: block;">
-        {% else %}
-        <div role="alert" class="status message submission-error" style="display: none;">
-        {% endif %}
-          <h3 class="message-title">{% trans "The following errors occurred while processing your registration: " %}</h3>
-          <ul class="message-copy">
-            {% if err_msg %}
-              <li>{{err_msg}}</li>
+<div id="password-reset-confirm-container" class="login-register">
+    <section id="password-reset-confirm-anchor" class="form-type">
+        <div id="password-reset-confirm-form" class="form-wrapper">
+            <div class="status submission-error {% if not err_msg %} hidden {% endif %}" aria-live="polite">
```

Comment:
I've noticed that the ```hidden```  class appears on the same ```<div>``` as the ```aria-live```.  This will not work in some browser/AT combinations.  The AT is programmed to watch containers and elements with aria-live attributes for changes.  However, it cannot watch a container that is not in the DOM.   aria-live regions should be present in the DOM at page load.  Child content can be dynamically added and removed.  Suggest wrapping this ```div``` in a parent ```div``` that has the aria-live attribute on it.  That way, the toggling of content takes place entirely within the empty parent div.